ui/shared/components/panel/variants/selectors.js

```diff
@@ -15,21 +15,41 @@ import {
 import {
   getVariantTagsByGuid, getVariantNotesByGuid, getSavedVariantsByGuid, getAnalysisGroupsByGuid, getGenesById, getUser,
   getFamiliesByGuid, getProjectsByGuid, getIndividualsByGuid, getRnaSeqDataByIndividual,
+  getPhenotypeGeneScoresByIndividual,
 } from 'redux/selectors'
 
-export const getRnaSeqOutilerDataByFamilyGene = createSelector(
+export const getIndividualGeneDataByFamilyGene = createSelector(
   getIndividualsByGuid,
   getRnaSeqDataByIndividual,
-  (individualsByGuid, rnaSeqDataByIndividual) => Object.entries(rnaSeqDataByIndividual).reduce(
-    (acc, [individualGuid, rnaSeqData]) => {
-      const { familyGuid, displayName } = individualsByGuid[individualGuid]
-      acc[familyGuid] = Object.entries(rnaSeqData.outliers || {}).reduce(
-        (acc2, [geneId, data]) => (data.isSignificant ?
-          { ...acc2, [geneId]: { ...(acc2[geneId] || {}), [displayName]: data } } : acc2
-        ), acc[familyGuid] || {},
-      )
-      return acc
-    }, {},
+  getPhenotypeGeneScoresByIndividual,
+  (individualsByGuid, rnaSeqDataByIndividual, phenotypeGeneScoresByIndividual) => (
+    Object.entries(individualsByGuid).reduce((acc, [individualGuid, { familyGuid, displayName }]) => {
+      const rnaSeqData = rnaSeqDataByIndividual && rnaSeqDataByIndividual[individualGuid]?.outliers
+      const phenotypeGeneScores = phenotypeGeneScoresByIndividual && phenotypeGeneScoresByIndividual[individualGuid]
+      if (!rnaSeqData && !phenotypeGeneScores) {
+        return acc
+      }
+      return {
+        ...acc,
+        [familyGuid]: {
+          rnaSeqData: Object.entries(rnaSeqData || {}).reduce(
+            (acc2, [geneId, data]) => (data.isSignificant ? {
+              ...acc2,
+              [geneId]: [...(acc2[geneId] || []), { ...data, individual: displayName }],
```

Comment:
its a little confusing to have this key be `individual`, it implies that it is the full individual object. `individualId` or `individualName` would be better

Reply:
Okay. Since it is the `displayName`, I'll use `individualName`.